docker/secondary-analysis-python/tools/secondary_analysis/secondary_analysis/create_analysis_json.py

```diff
@@ -123,9 +123,9 @@ def main():
     parser.add_argument('-run_type', required=True, help='Should always be "run" for now, may be "copy-forward" in some cases in future')
```

Comment:
This doesn't matter that much, but the convention I normally see in python is `--long-argument` and `-a` for short single-letter arguments.